### PR TITLE
Missing translation terms in templates

### PR DESCRIPTION
**Pull request by kryskool**

### CODE REVIEW

File: tcms/templates/plan/get_cases.html [hunk 1]
```diff
@@ -16,38 +16,38 @@
...
                                     <ul>
-                                        <li><input type="button" class="new_run icon_plan" value="Create new TestRun" id="js-new-run" data-param="{% url "testruns-new" %}" /></li>
+                                        <li><input type="button" class="new_run icon_plan" value="{% trans "Create new TestRun" %}" id="js-new-run" data-param="{% url "testruns-new" %}" /></li>
                                     </ul>
                                 </li>
                         {% endif %}
 			{% if perms.testcases.change_testcase %}
                                 {% if REQUEST_CONTENTS.template_type == 'review_case' %}
-                                <li><input type="button" value="Reviewer"       title="Change TestCase reviewer" class="btn_reviewer       assignee sprites icon_plan" /></li>
+                                <li><input type="button" value="Reviewer" title="Change TestCase reviewer" class="btn_reviewer assignee sprites icon_plan" /></li>
```
Comment (atodorov) [inline tcms/templates/plan/get_cases.html]: Missing trans tag here. Probably you want to add it?

File: tcms/templates/plan/get_cases.html [hunk 2]
```diff
@@ -16,38 +16,38 @@
...
                                 {% else %}
-                                <li><input type="button" value="Default Tester" title="Change default tester"    class="btn_default_tester assignee sprites icon_plan" /></li>
+                                <li><input type="button" value="Default Tester" title="Change default tester" class="btn_default_tester assignee sprites icon_plan" /></li>
```
Comment (atodorov) [inline tcms/templates/plan/get_cases.html]: also missing trans tag

File: tcms/templates/plan/get_cases.html (hunk 1 above)
Comment (kryskool) [reply tcms/templates/plan/get_cases.html]: Done

File: tcms/templates/plan/get_cases.html (hunk 2 above)
Comment (kryskool) [reply tcms/templates/plan/get_cases.html]: Done

**Pull request merged**